photon-client/src/main/scala/com/linkedin/photon/ml/data/avro/README.md

```diff
@@ -0,0 +1,3 @@
+Refactorings needed here:
+- Simplify avro field names, stop having different names "label" and "response"
```

Comment:
Actually, the plan is to get rid of it entirely, and allow user to pass in the "label" column name.